docs/doc/60-contributing/03-rfcs/20220530-user-friendly-releases.md

```diff
@@ -0,0 +1,71 @@
+---
+title: User-friendly Releases
+description: RFC for user-friendly releases
+---
+
+# Summary
+
+A simple user-friendly release solution.
+
+# Motivation
+
+In the past, only specific binaries were included in the Databend release. Understanding how to operate and maintain Databend often requires extensive discussion.
+
+This proposal aims to improve this process by providing some simple tips and default configs to help make the process painless for users.
+
+# Detailed design
+
+The release directory structure is roughly as follows:
+
+```rust
+.
+├── bin
+│   ├── databend-query
+│   ├── databend-meta
+│   └── databend-metactl (?)
+├── configs
+│   ├── databend-query.toml
+│   └── databend-meta.toml
+├── docs 
+│   ├── basic-guide.md
+│   ├── faq.md
+│   └── mini-tutorial.md (?)
+├── data/ (?)
```

Comment:
Putting data in releases is not a good idea for me.

Even if we put them in data, users without tutorials will not know how to use them. But if we have tutorials, why not let them download directly...

Reply:
It's actually more like some actual runnable query examples.

It appears in  [cayley](https://github.com/cayleygraph/cayley/releases/) and is listed in the proposal only because it looks interesting. For Databend, there may indeed be no value.

Reply:
I have removed the description of `data/`.